pkg/backend/httpstate/backend.go

```diff
@@ -915,7 +916,7 @@ func (b *cloudBackend) DoesProjectExist(ctx context.Context, orgName string, pro
 	}
 
 	getDefaultOrg := func() (string, error) {
-		return pkgWorkspace.GetBackendConfigDefaultOrg(nil)
+		return placeholder.GetDefaultOrg(ctx, b, nil)
```

Comment:
Functionally, what does a `nil` project mean?

Reply:
One of the places you can get default org is from the project; some places there is no project so it's one less place to look to see if there's a default org configured.